run_galaxy_workflow.py

```diff
@@ -68,6 +66,10 @@ def get_args():
                             default=None,
                             help="Yaml file with allowed steps that can have errors."
                             )
+    arg_parser.add_argument('-s', '--state-file',
```

Comment:
So, will this automatically resume, or will the logic that runs this script need to pass the state file explicitly after the first run?

Reply:
By default it will try to resume if it finds the file in place. For the design that you have, since I presume that working directories get deleted, you might want to pass a file here that is elsewhere and doesn't get deleted in-between disconnections. I could imagine this causing problems for unaware users running things in the same working directory though.